spec/index.bs

```diff
@@ -3772,22 +3786,17 @@ GPUTexture includes GPUObjectBase;
     : <dfn>usage</dfn>
     ::
         The allowed usages for this {{GPUTexture}}.
-</dl>
-
-{{GPUTexture}} has the following internal slots:
-
-<dl dfn-type=attribute dfn-for=GPUTexture>
-    : <dfn>\[[size]]</dfn>, of type {{GPUExtent3D}}
```

Comment:
Clarification note for future generations: this removal is intentional, and has been replaced by direct references to the texture width/height/depthOrArrayLayers